redbot/cogs/economy/economy.py

```diff
@@ -113,8 +115,7 @@ def __init__(self, argument):
             self.operation = "set"
         else:
             raise RuntimeError
-
-
```

Comment:
You removed a line in here and added whitespace to the other line which makes this PR fail the style check. You need to ensure that there are two empty lines between the class definitions. Or you can run Black to reformat this for you as explained in [our contributing guidelines](https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/CONTRIBUTING.md)